dandi/cli/cmd_organize.py

```diff
@@ -212,6 +235,40 @@ def _get_metadata(path):
 
     metadata = create_unique_filenames_from_metadata(metadata)
 
+    # update metadata with external_file information:
+    external_files_missing_metadata_bool = [
+        len(m["external_file_objects"]) == 0 for m in metadata
+    ]
+    if all(external_files_missing_metadata_bool) and rewrite == "external-file":
+        lgr.warning(
+            "rewrite option specified as 'external_file' but no external_files found"
+            "linked to any nwbfile found in %s",
+            paths,
+        )
+
+    elif not all(external_files_missing_metadata_bool) and rewrite != "external-file":
+        raise ValueError(
+            "rewrite option not specified but found external video files linked to "
+            f"the nwbfiles "
+            f"""{[metadata[no]['path']
```

Comment:
Use `%s` log formatting here.

Reply:
Do you suggest we log this Exception before throwing it?
Or to use format method instead of f strings: `'string{}'.format(var)`

Reply:
Oh, sorry, for some reason I thought this was a logging call.  Never mind.